src/EditCommand.php

```diff
@@ -5,6 +5,7 @@
 use Symfony\Component\Process\Process;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
```

Comment:
It's not part of PSR-2, it's part of Laravel's code style (which is a modified PSR-2 afaik) Other Laravel repos (like framework) [have a .php_cs file](https://github.com/laravel/framework/blob/5.1/.php_cs) that will tell phpcs how to style code.